pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -209,12 +210,16 @@ protected String getQuery(SlingHttpServletRequest request) {
         SimpleDateFormat sdf = new SimpleDateFormat("MMM dd, yyyy HH:mm");
         
         //logic for file name is present in ModuleVersionUpload.java
-        if(draftMetadata.isPresent() && draftMetadata.get().moduleType().get()!=null){
-            m.put("moduleType",draftMetadata.get().moduleType().get());
-        }else if(releasedMetadata.isPresent() && releasedMetadata.get().moduleType().get()!=null){
-            m.put("moduleType",releasedMetadata.get().moduleType().get());   
-        }else{
-            m.put("moduleType","-");
+        if ("pantheon/assembly".equals(resource.getResourceType())) {
```

Comment:
The string values can be made into constants. As we are already doing at other places, this will make code consistent.